xla_native_functions.yaml

```diff
@@ -178,10 +182,7 @@ supported:
   - log10
   - log_sigmoid_backward
   - log_sigmoid_forward
-  - logical_and
-  - logical_not
-  - logical_or
-  - logical_xor
+  - logdet
```

Comment:
why is logdet here? Maybe rebase this pr?

Reply:
Updated